packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx

```diff
@@ -35,34 +35,45 @@ const Text = styled.div`
 
 interface CoinmarketPaymentTypeProps {
     children?: React.ReactNode;
-    method?: BuyCryptoPaymentMethod | SavingsPaymentMethod;
+    method?: BuyCryptoPaymentMethod | SellCryptoPaymentMethod | SavingsPaymentMethod;
+    methodName?: string | undefined;
```

Comment:
nitpick, `?` and `| undefined` is the same
```suggestion
    methodName?: string;
```

Reply:
so my change is cleaner or not? At least shorter?